docs/source/02-Analytics-Query/Funnel-View/Sign-Up-Funnel.sql

```diff
@@ -0,0 +1,334 @@
+/*
+**Question**:
+
+There are 8 primary phases in login.gov sign-up flow that the user is from a service provider.
+
+1. User has entered a valid email
+2. User has entered a confirmation email
+3. User has created a valid password
+4. User has visited the MFA setup page
+5. User has finished MFA setup
+6. User has viewed personal key page
+7. User has visited hand-off page with service provider
+8. Sign-up workflow is finished, user is on service provider website
+
+We want to know how many unique sessions are there in each phase.
+
+**Definition**:
+
+1. User has entered a valid email::
+
+    if have 'User Registration: Email Submitted' and success = True
+        return True
+
+2. User has entered a confirmation email::
```

Comment:
before you had said that this is potentially a different session - so looking at the initial data- can you pull what it would look like if the `'User Registration: Email Submitted'` is not part of the same session as `'User Registration: Email Confirmation'`. ie. - and for all the other events - 2-8, remove the condition of `(have 'User Registration: Email Submitted' and success = True)`?
```
if (have 'User Registration: Email Confirmation' and success = True)
        return True
```

Reply:
I checked the sample data, they are in the same session.

I also checked with Chrome built-in network capture developer tools, I think a session token is created at the email entered, then chrome will find that token even they closed the chrome window. I think other web browsers might be the same.

So, In conclusion, they are in the same session.